src/references/services.md

```diff
@@ -0,0 +1,75 @@
+# Services
+
+Most real world applications work together with other services to store data, logs, to exchange messages with other systems etc.
+The most common example is probably a database. There are at least 2 ways to connect an Epinio application to a service.
+The first one is to provision the service manually and [create an Epinio Configuration](configurations.md) to make the service connection details
+available to the application.
+
+The second one lets Epinio provision the service which you can later bind to your application with the `epinio service bind` command. This page explains how this works.
+
+Services in Epinio are managed by the [epinio service](cli/epinio_service.md) commands.
```

Comment:
I got 404 here, I think you forgot to add the link into the SUMMARY.

Reply:
You can add this in the SUMMARY just under the configurations part:

```
      - [epinio_service](references/cli/epinio_service.md)

        - [epinio_service_bind](references/cli/epinio_service_bind.md)
        - [epinio_service_catalog](references/cli/epinio_service_catalog.md)
        - [epinio_service_create](references/cli/epinio_service_create.md)
        - [epinio_service_delete](references/cli/epinio_service_delete.md)
        - [epinio_service_list](references/cli/epinio_service_list.md)
        - [epinio_service_show](references/cli/epinio_service_show.md)
```

Reply:
Right! Fixed in d783815